core/src/main/java/com/gamification/core/controllers/BaseController.java

```diff
@@ -25,7 +28,8 @@
     protected abstract Class<M> getMapper();
 
     @PostMapping(value = RestAddress.SAVE)
```

Comment:
i think there is an annotation that can be over method to explain its functionality please add it to.